apiserver/failoverservice/failoverimpl.go

```diff
@@ -105,7 +106,7 @@ func QueryFailover(name, ns string) msgs.QueryFailoverResponse {
 		Status:  msgs.Status{Code: msgs.Ok, Msg: ""},
 	}
 
-	_, err = validateClusterName(name, ns)
+	cluster, err := validateClusterName(name, ns)
```

Comment:
I believe the `var err` in line 103 can now be removed :smile:!

Reply:
`var err error` has been removed.